### ISSUE TITLE
Unable to execute on Alpine 3.12

### ISSUE THREAD

**Issue by pickgr** [instaclustr/terraform-provider-instaclustr#63]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

terraform 12.28

### Affected Resource(s)

* instaclustr_1.7.3

### Debug Output

exit status 1: running "tf-plan dev" in "/home/atlantis/.atlantis/repos/seismic/terraform/1280/default/providers/instaclustr/azure-dev/westus/dev-eventbus": 

Error: Failed to instantiate provider "instaclustr" to obtain schema: fork/exec /home/atlantis/.terraform.d/plugins/linux_amd64/terraform-provider-instaclustr_v1.7.3: no such file or directory

No workflow (dev) tfvars file to include.
Loading the workspace (default) tfvars file!

### Expected Behavior

terraform init should instantiate the provider

### Actual Behavior

See the error above.

### Steps to Reproduce

This running on an Alpine 3.12 container.  Specifically, we use the open source atantis project to run our terraform

https://github.com/runatlantis/atlantis/blob/master/docker-base/Dockerfile#L4

1. `terraform init`

### Important Factoids

We have verified that the binary is installed in the correct place even though the error says "no such file or directory"

The binary we are trying to use is from 

https://github.com/instaclustr/terraform-provider-instaclustr/releases/download/v1.7.3/terraform-provider-instaclustr_v1.7.3_linux_amd64.zip

We have 2 other plugins (ibm and kafka) that run fine.  Note that only instaclustr shows dynamic linking below"

```
terraform-provider-ibm_v1.20.1:        ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=GCMn5HhuuXZOqpY21QlD/_L0vEtS3sXKKnsXKOFG4/j1FUonfQVGaR1e8uBwwO/-eepCW4-Kjoi2Vvs-3PH, stripped
terraform-provider-instaclustr_v1.7.3: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=M5Xdc1ExCGvspcOpspb3/BJPLKjMZsw9nuGLdaoOc/YU3a06q9LCwVmcH6unYm/u6A07R8DVuXAdy9zvBSs, not stripped
terraform-provider-kafka:              ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=-KVZ0THLa8TLXf2dGDB_/SP5IHMHjcYS2BxFe2YcG/-IeImQNtY6Uq4O_g64Qx/pnfiI4NpyVjQ_z5y4CWO, stripped

```
It may be something similar to 

https://stackoverflow.com/questions/36279253/go-compiled-binary-wont-run-in-an-alpine-docker-container-on-ubuntu-host

**Comment by toddschilling** [comment]
I've tested this using the `runatlantis/atlantis` docker image locally.  Building the provider with `CGO_ENABLED=0` does the trick.  I'll generate a fork and submit a PR later today to add this to the CircleCI releases.

**Comment by pickgr** [comment]
https://github.com/instaclustr/terraform-provider-instaclustr/pull/64

**Comment by Chrismelba-Instaclustr** [comment]
Looks like this is complete and merged now. Are we good to close this?

**Comment by peter-toms** [comment]
Closing this one off as @toddschilling added Alpine support in #64, thanks!

**Comment by toddschilling** [comment]
> Closing this one off as @toddschilling added Alpine support in #64, thanks!

There is still an open support ticket on this.  My local builds work on the atlantis docker image but the release version on github does not.  I've submitted my test code to support@instaclustr.com.

**Comment by peter-toms** [comment]
Hi @toddschilling adding internal reference here for that issue thanks for raising!  Cheers INS-12933

**Comment by toddschilling** [comment]
@peter-toms @pickgr I've forked this repo and recreated the CircleCI pipeline on my own account.  When built locally, a 33.2 MB provider is generated and that provider works on the Atlantis alpine image.  However, when build on the circlcleci/golang:1.14 docker container, with or without the `-tags netgo` parameter, a 33.3 MB provider is generated that does not work on Atlantis.  I'm running additional tests now.

**Comment by toddschilling** [comment]
@peter-toms I've just opened a pull request to fix this.  https://github.com/instaclustr/terraform-provider-instaclustr/pull/77.  As @pickgr showed above, other well known providers that we're using are also static and stripped.  I can also add `-ldflags '-w -s -extldflags "-static"'`.

**Comment by pickgr** [comment]
Thanks @toddschilling  !

**Comment by hail-insta** [comment]
After some internal discussions, we have decided to build the standard linux binary with static linking to include required packages within the binary to support compatibility. This means that the alpine and linux builds will be identical and the alpine build will still remain in our release, but we will likely deprecate it in the future.
Thanks @toddschilling - we will still use your change for the build. Appreciate the suggestion on stripping and DWARF debug removal as it has significantly reduced the binary size

**Comment by hail-insta** [comment]
This has been addressed. Please find the fixed binary in release v1.9.4 - tested that both Linux and Alpine assets can be used for atlantis image!
